integration-tests/src/test/resources/scripts/stop.sh

```diff
@@ -0,0 +1,3 @@
+sudo systemctl stop constellation && \
+  sudo rm -rf /home/admin/constellation/tmp && \
+  sudo rm -f /home/admin/constellation/logs/app.log
```

Comment:
I guess we need to remember to not run stop.sh as a part of shutting down the cluster. For sure not before sending the logs to s3.

Reply:
Well, @marcinwadon had an opposite comment. Initially, I didn't clean up the logs and he suggested that I should do it.

Reply:
I mean that, we have a step that sends the logs to s3 as a part of the workflow. If after the test we first run the stop.sh script then we will send nothing to the s3 bucket.

Reply:
We run `start.sh` after `stop.sh` so we will send logs from the restart of the node.

Reply:
@TheMMaciek is right, we need all logs from the execution. Originally I thought you will be reusing logs, so removal step was important, to keep logs only from the current run. By current run I mean the whole single execution on GH. Since you are starting an infra and destroying it every time, there is no need to remove logs using the script.

PS. Remember also about local executions as I saw you are using such.

Reply:
I updated the PR, with keeping the logs. @marcinwadon  By local executions, you mean stuff that is run by CI node? These logs are stored in the CI itself.

Reply:
Nope, the executions you ran locally from your pc. You're not destroying the infra every time so logs are aggregating all the time, hence you will be unable to easily search for errors